src/components/interface/Select/SearchableSelect.js

```diff
@@ -20,6 +20,7 @@ const SearchableSelect = ({
   selected,
   filter,
   onChange,
```

Comment:
What is th difference beween `onChange` and  `onTextChange`?

Reply:
onChange only give you a result when you select an item 
onTextChange is plugged to the textBox